libsolidity/ast/Types.cpp

```diff
@@ -2273,27 +2309,6 @@ set<string> StructType::membersMissingInMemory() const
 	return missing;
 }
 
-bool StructType::recursive() const
```

Comment:
Ah, that's unfortunate. I'd probably need this function for https://github.com/ethereum/solidity/pull/6282. Is it possible to split it out again?

Reply:
Can you just use `interfaceType()` directly? It kinda does that check implicitly

Reply:
I could offer a dedicated function that uses `interfaceType()` to gather that information I suppose

Reply:
like, `interfaceType()` would set an `boost::optional<bool> m_recursive;`

Reply:
Having a flag that signals if this struct declares a member of the same type e.g. `boost::optional<bool> m_recursive` would be enough I guess.